config/database.yml

```diff
@@ -16,13 +16,10 @@
 #
 default: &default
   adapter: postgresql
-  encoding: unicode
```

Comment:
Don't accept the changes in this file. It was changed so it works locally for me.